### PR TITLE
Add official support to npm

### PR DESCRIPTION
**Pull request by rafaelfranca**
Address #423 and #379.

Please review @justin808 @callumacrae.

I hope you don't mind we remove ownership access to the package in npmjs since we prefer to keep release access only to the core team.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ Require both `jquery` and `jquery_ujs` into your application.js manifest.
 //= require jquery_ujs
 ```
 
+Installation using npm.
+------------
+
+Run `npm install jquery-ujs --save` to install the jquery-ujs package.
```
Comment (justin808) [inline README.md]: I usually do the option --save before the package name, FWIW.

**Comment by justin808** [comment]
LGTM, although I didn't try the commands.

**Comment by guilleiguaran** [comment]
👍

**Comment by callumacrae** [comment]
:+1: 🎉

**Pull request merged**